feature_engine/features/feature_extraction.py

```diff
@@ -0,0 +1,520 @@
+import os
```

Comment:
Note: can we rename this file to be descriptive of what's inside? This isn't just about feature extraction; this is specifically a helper file for politeness_v2. Can we rename this something like `politeness_v2_helper`?